src/main/java/org/broadinstitute/barclay/argparser/ClassFinder.java

```diff
@@ -0,0 +1,188 @@
+package org.broadinstitute.barclay.argparser;
+
+import org.apache.logging.log4j.Logger;
```

Comment:
This will help if https://github.com/broadinstitute/gatk/issues/2176 is implemented, but also for make this project more general.